### PR TITLE
SRVCOM-1288 + SRVCOM-1664: Clean up attributes, prereqs

### PR DESCRIPTION
**Pull request by abrennan89**
Jiras: https://issues.redhat.com/browse/SRVCOM-1288 + https://issues.redhat.com/browse/SRVCOM-1664

Applies for OCP 4.6+

Direct preview: https://deploy-preview-41667--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-pingsource.html

### CODE REVIEW

File: modules/serverless-pingsource-yaml.adoc [hunk 1]
```diff
@@ -1,28 +1,23 @@
 // Module included in the following assemblies:
 //
-// * serverless/event_sources/serverless-pingsource.adoc
+// * /serverless/develop/serverless-pingsource.adoc
 
 :_content-type: PROCEDURE
 [id="serverless-pingsource-yaml_{context}"]
-= Using a ping source with YAML
+= Creating a ping source by using YAML
```
Comment (abrennan89) [inline modules/serverless-pingsource-yaml.adoc:7]: Note to self: update topicmap so left nav matches this

**Review by aliok — APPROVED** [review]
/lgtm

All good techincally

File: serverless/develop/serverless-pingsource.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="serverless-pingsource"]
-= Using a ping source
+= Creating a ping source
```
Comment (jrangelramos) [inline serverless/develop/serverless-pingsource.adoc:3]: Same question here.. shouldn't the left title be the same as the title here?

**Review by jrangelramos — COMMENTED** [review]
just one minor question.. other than that, it looks good to me.

**Pull request merged**